### PR TITLE
#71 & #225 engine kill and capship parts

### PR DESCRIPTION
**Pull request by FireFlyEx**
Changed miners, liners, transports, and trains to no longer have free engine kill. They now have partial engine kill. **Used the Engine (1/8) and Ship (7/8) rule.**

Changed capital ship collision groups to no longer activate fuses that would destroy the ship if they were destroyed. Also, disabled health root proxy so that group hp does not affect the main body. These changes allow parts to be blown off the ship and the ship is still flyable (unless the engines get destroyed..).

**NEXT -=-=-=-**

Update all capital ship fuses to give them proper destruction.

### CODE REVIEW

**Comment by erikroe** [comment]
@FireFlyEx My mistake for having been unprecise on the issue: The 1:8 rule is good for the transports and trains. For anything bigger like the Liners and Miner I would like them to be completely without engine kill. This is simply to make it consistent with the battleships and cruisers.
Little background why we do this at all: Engine kill basically increases the damage of the already long-firing artillery guns a lot if you just drop from cruise. I don't want that, because it makes them have ridiculous-long ranges ;)

**Comment by erikroe** [comment]
@FireFlyEx On the Collision Groups: Please do not comment out the fuses. The destruction and burning of the ship by having its collision groups being destroyed is actually intended behaviour. It should make people aim more tactically to weaker spots. And removing those fuses would lead to your ship e.g. losing its engine and you as player can never get anywhere again and are helpless. The burning fuses should be there always anyway.

File: DATA/SHIPS/shiparch.ini [hunk 1]
```diff
@@ -1206,7 +1206,7 @@ DA_archetype = ships\utility\luxury_liner\luxury_liner.cmp
 material_library = ships\utility\utility_liner.mat
 material_library = fx\envmapbasic.mat
 envmap_material = envmapbasic
-linear_drag = 1
+linear_drag = 8751
```
Comment (erikroe) [inline DATA/SHIPS/shiparch.ini:1209]: The 1 at the end can be just made to a 0, too. Looks more clean.

**Comment by FireFlyEx** [comment]
> 

Why aren't we deleting the old engines, etc, that we aren't using?

**Comment by FireFlyEx** [comment]
> @FireFlyEx On the Collision Groups: Please do not comment out the fuses. The destruction and burning of the ship by having its collision groups being destroyed is actually intended behaviour. It should make people aim more tactically to weaker spots. And removing those fuses would lead to your ship e.g. losing its engine and you as player can never get anywhere again and are helpless. The burning fuses should be there always anyway.

The engine groups themselves could be made invulnerable to solve that problem, or the engines only could have a special fuse that destroys the ship if they are destroyed. Does it have to be every part on the ship that makes it quicker to kill? I feel like engines as a vulnerability makes sense but not anything else. A ship should be able to survive if it loses the wings or the tail.

File: DATA/SHIPS/shiparch.ini (hunk 1 above)
Comment (FireFlyEx) [reply DATA/SHIPS/shiparch.ini:1209]: Done

**Comment by erikroe** [comment]
NPC engines should not be touched. But there is more player engines.
The older engines are still there because we first need to make sure they are not used anymore by any other NPCs somewhere. That's another task.

**Comment by erikroe** [comment]
I suggest first we leave the fuses as they were before. We can still talk about this later when checking specifically per capship what pieces could be made destroyable without taking the entire ship down.

**Comment by FireFlyEx** [comment]
> NPC engines should not be touched. But there is more player engines. The older engines are still there because we first need to make sure they are not used anymore by any other NPCs somewhere. That's another task.

Where are these engines? I only see 2 and I've changed those.

- cv_engine_liner01
- cv_engine_miner01

**Comment by erikroe** [comment]
Liner and Miner only exist for CV, that is correct.

**Comment by FireFlyEx** [comment]
> Liner and Miner only exist for CV, that is correct.

But what other player engines are you wanting me to change?

**Comment by erikroe** [comment]
Engines like rh_engine_train01 etc. Trains and Transports have these engines, too. Same for KU, BR, LI, PI

**Comment by FireFlyEx** [comment]
All good now.

**Review by erikroe — APPROVED** [review]

**Pull request merged**